viral/basic_plots.py

```
@@ -0,0 +1,938 @@
+    axes[0].set_ylabel("Trials with licks in AZ (%)")
+    axes[1].set_title("After Learning")
+    axes[0].set_title("Before Learning")
+    sns.despine()
+    plt.tight_layout()
+
+    plt.rcParams["pdf.fonttype"] = 42
+    plt.savefig(
+        SERVER_PATH
+        / "viral_plots"
+        / "anticipatory_licking"
+        / f"basic_anticipatory_licking_plot.pdf",
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)